src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -0,0 +1,166 @@
+<style>
+#open-settings {
+    color: var(--text-muted);
+    display: block;
+    margin: 15px 15px 5px;
+}
+
+.modal-wrap {
+    -webkit-transition: all 0.3s ease-in-out;
```

Comment:
IMO, you can drop the vendor prefix here. All modern browsers support it unprefixed (and otherwise, it should still stay just before the unprefixed property IMO).